docs/extras/project-ideas/index.md

```diff
@@ -0,0 +1,176 @@
+
+#### How
+
+Using the [OpenTrials query interface](https://query.opentrials.net/), build a set of queries designed for outlier detection across key data fields.
+
+- [A simple introduction to outlier detection with SQL](https://www.periscopedata.com/blog/outlier-detection-in-sql.html)
+
+#### Skills
+
+- Intermediate SQL
+
+### Visualisation of discrepancies
+
+#### What
+
+By collecting a significant amount of publicly available data on clinical trials, we've found that different records for the same trial often have different information. Such differences can be, for example, in the supposed number of participants who took part in a trial, whether the trial is still ongoing, or if the trial's results have been published.
+
+These discrepancies are important as all these public sources of trial data can be taken as "official", and referenced in the academic literature, for example. Therefore, exposing information on discrepancies is an important outcome of building the OpenTrials database.
+
+#### How
+
+Use the [OpenTrials API](http://api.opentrials.net/v1/docs/) to find trials with detected discrepancies. Look for common types of discrepancies, and build basic visualisations based on type (e.g.: the publication date or participation count) or frequency (e.g. are discrepancies becoming more common or less common?).
+
+#### Skills
+
+- Intermediate JavaScript and HTML
+- Basic knowledge of visualisations
+
+### Visualise publication lag
+
+#### What
+
+It is imperative that results of clinical trials are published. This is how knowledge is shared: about the effectiveness, or not, of a particular intervention for a particular condition, for example. Unfortunately, many trials have their results published with a long delay, or, simply do not have their results published at all.
+
+#### How
+
+Use the [OpenTrials API](http://api.opentrials.net/v1/docs/) to find trials with a large lag in publication of results, or, trials that have finished and have not yet published results. Build basic visualisations that show lag over time, or lag by condition, by treatment, by trial sponsor, by primary investigator and so on.
+
+#### Skills
+
+- Intermediate JavaScript and HTML
+- Basic knowledge of visualisations
+
+### Visualise the relationship between conditions and unpublished/published/ongoing trials
+
+#### What
+
+Certain conditions can be subject to clinical trials at different frequencies over time. Of course, there can be very good reasons for this, for example, a boost in clinical trials for Ebola around the Ebola crisis of 2014. Choose one or several conditions, and show information on a timeline for ongoing trials, trials that have finished and have published results, and trials that have finished yet do not have published results.
```

Comment:
Similar to a previous comment, we just have the registration date, so it's a bit hard to do a timeline. We can, however, see the relationship between conditions and trial status, or how long a trial has been registered.